crates/cxx-qt-lib/src/core/mod.rs

```diff
@@ -110,6 +113,10 @@ mod qvector;
 pub use qvector::{QVector, QVectorElement};
 
 mod qtlogging;
+mod qobject;
+
+pub use qobject::QObjectExt;
```

Comment:
try to keep these grouped and ordered alphabetically, so should probably be higher up?